NetcodePatcher.Tools.Common/PatcherLoadContext.cs

```diff
@@ -27,7 +28,19 @@ public PatcherLoadContext(string name, PatcherConfiguration configuration) : bas
         if (assemblyName.Name is null) return null;
 
         if (SharedDependencyAssemblyNames.Contains(assemblyName.Name))
-            return Default.LoadFromAssemblyName(assemblyName);
+        {
+            string? sharedPath = ResolveAssemblyToPath(assemblyName);
+            if (sharedPath is null)
+            {
+                Log.Debug("Shared dependency {SharedName} not found in {CommonDir} or {SharedDir}, trying to load from system", assemblyName, _configuration.PatcherCommonAssemblyDir, _configuration.PatcherNetcodeSpecificAssemblyDir);
+                return Default.LoadFromAssemblyName(assemblyName);
+            }
+            else
+            {
+                Log.Debug("Shared Dependency {SharedName} loading from {Directory}", assemblyName, sharedPath);
+                return LoadFromAssemblyPath(sharedPath);
```

Comment:
Although the shared path has been resolved in a non-standard manner, serilog should still be loaded into the `Default` context - please try `Default.LoadFromAssemblyPath(sharedPath);`

Reply:
that works as expected.